OpenRA.Game/World.cs

```diff
@@ -387,6 +387,7 @@ public void Dispose()
 
 			frameEndActions.Clear();
 
+			Sound.StopAudio();
```

Comment:
I just noticed that this cuts off the "battlecontrol terminated" sound. It's now just "battlecontrol terminat..."